api-stat.go

```diff
@@ -49,6 +49,31 @@ func (c Client) BucketExists(bucketName string) (bool, error) {
 	return true, nil
 }
 
+// List of header keys to be filtered, usually
+// from all S3 API http responses.
+var defaultFilterKeys = []string{
+	"Transfer-Encoding",
+	"Accept-Ranges",
+	"Date",
+	"Server",
+	"Vary",
+	"x-amz-request-id",
```

Comment:
looks like minio server sends `X-Amz-Request-Id`

```
mc: <DEBUG> HTTP/1.1 200 OK
Content-Length: 2248
Accept-Ranges: bytes
Content-Type: application/octet-stream
Date: Tue, 13 Dec 2016 21:35:29 GMT
Etag: "1ff633d084da03ff72793c7a93787fca"
Last-Modified: Tue, 13 Dec 2016 21:31:20 GMT
Server: Minio/DEVELOPMENT.GOGET (linux; amd64)
Vary: Origin
X-Amz-Request-Id: O1BCY6HBCDTU0MY0
```

don't know about x-amz-id-2

it will be safer to make it canonical somewhere